dabplus-on-air-processing.spec

```diff
@@ -11,7 +11,7 @@ Name:     dabplus-on-air-processing
 BuildRequires: make
 BuildRequires: systemd
 
-Requires: liquidsoap
+Requires: liquidsoap = ${_ls_version}
```

Comment:
[liquidsoap 1.3.0](https://github.com/savonet/liquidsoap/releases/tag/1.3.0) wasn't backwards compatible at all (search for "Legacy code must be updated" in the release notes). It looks like the RELEASING docs are a bit outdated.

The version needs to be exact since liquidsoap always expects it's libs to be in [the liqdir folder](https://github.com/hairmare/dabplus-on-air-processing/blob/4c11283146f810a0a79854a6f6a4939726677c7a/dabplus-on-air-processing.spec#L7) which is version specific. 

Right now this is already broken due to us having liquidsoap 1.3.0 installed but the old version of this package still installing it's content to the 1.2.1 dir.